vanilla-framework/vanilla-framework-react
# Add eslint to app

## Done

- Add Eslint to app

## QA

- Pull branch
- Run `npm run lint`
- There should be three errors* related to `react/prop-types`

(* These errors will be resolved in #8)
